domains/jacksonjperry03.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "jacksonjperry03",
+           "email": "jackson.perry@uri.edu",
+           "discord": "395990222833451008"
+        },
+    
+        "record": {
+            "CNAME": "https://jacksonjperry03.github.io/jackson-perry-portfolio/"
```

Comment:
Invalid CNAME record. CNAME records should be a single string with a hostname. A hostname contains no protocols or URL paths.

Reply:
dont use protocols, (https://)  cname also noy path too
jst use this 
jacksonjperry03.github.io

Reply:
I made another commit.